amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2609,6 +2628,18 @@ void fillDescriptionPtrfromDims(RpptDescPtr &descPtr, vxTensorLayout layout, siz
     }
 }
 
+void fillAudioDescriptionPtrFromDims(RpptDescPtr &descPtr, size_t *tensorDims) {
+    descPtr->n = tensorDims[0];
+    descPtr->h = tensorDims[2];
+    descPtr->w = tensorDims[1];
+    descPtr->c = 1;
+    descPtr->strides.nStride = descPtr->c * descPtr->w * descPtr->h;
+    descPtr->strides.hStride = descPtr->c * descPtr->w;
+    descPtr->strides.wStride = descPtr->c;
+    descPtr->strides.cStride = 1;
+    descPtr->numDims = 4;
```

Comment:
Is numDims always going to be 4?

Reply:
yes Shobana

Reply:
We have 4 dims in audio with last dim as 1 always. Thats how all the augmentations have been handled